Mathlib/Algebra/MvPolynomial/Equiv.lean

```diff
@@ -414,7 +413,7 @@ theorem coeff_eval_eq_eval_coeff (s' : Fin n → R) (f : Polynomial (MvPolynomia
   simp only [Polynomial.coeff_map]
 
 theorem support_coeff_finSuccEquiv {f : MvPolynomial (Fin (n + 1)) R} {i : ℕ} {m : Fin n →₀ ℕ} :
-    m ∈ (Polynomial.coeff ((finSuccEquiv R n) f) i).support ↔ Finsupp.cons i m ∈ f.support := by
+    m ∈ (Polynomial.coeff (finSuccEquiv R n f) i).support ↔ Finsupp.cons i m ∈ f.support := by
```

Comment:
```suggestion
    m ∈ ((finSuccEquiv R n f).coeff i).support ↔ Finsupp.cons i m ∈ f.support := by
```
untested. Maybe `m.cons i` too on the rhs?

Reply:
Whoops sorry I thought I'd gotten them all